Assets/Scripts/Item.cs

```diff
@@ -26,6 +28,10 @@ public class Item : MonoBehaviour {
 
   void Start() {
     originalScale = transform.lossyScale;
+    originalLayer = gameObject.layer;
+    foreach(Transform transform in GetComponentsInChildren<Transform>()) {
```

Comment:
This could just be `foreach(Transform childTransform in transform)`, no need for `GetComponentInChildren`